tool/site/client-react/src/components/prescription-tracker/prescriptionSummaryList.tsx

```diff
@@ -88,7 +88,10 @@ export const PrescriptionSummaryList: React.FC<PrescriptionSummaryProps> = ({
             <StyledPrescriptionActions prescriptionId={id} release />
           }
           {status === "With Dispenser" &&
-            <StyledPrescriptionActions prescriptionId={id} releaseReturn />
+            <>
+              <StyledPrescriptionActions prescriptionId={id} releaseReturn />
+              <StyledPrescriptionActions prescriptionId={id} withdraw />
+            </>
           }
         </SummaryList.Value>
```

Comment:
This is a released status rather than dispensed, the dispensing statuses a withdraw would be applied to include:

{(status === "With Dispenser - Active"  || status === "Dispensed") &&
            <>
              <StyledPrescriptionActions prescriptionId={id} withdraw />
            </>
          }

Reply:
refactored to use the above code snippet